tableauserverclient/server/endpoint/data_acceleration_report_endpoint.py

```diff
@@ -0,0 +1,30 @@
+from .endpoint import api, Endpoint
+from .permissions_endpoint import _PermissionsEndpoint
+from .default_permissions_endpoint import _DefaultPermissionsEndpoint
+
+from ...models.data_acceleration_report_item import DataAccelerationReportItem
+
+import logging
+
+logger = logging.getLogger('tableau.endpoint.data_acceleration_report')
+
+
+class DataAccelerationReport(Endpoint):
+    def __init__(self, parent_srv):
+        super(DataAccelerationReport, self).__init__(parent_srv)
+
+        self._permissions = _PermissionsEndpoint(parent_srv, lambda: self.baseurl)
+        self._default_permissions = _DefaultPermissionsEndpoint(parent_srv, lambda: self.baseurl)
+
+    @property
+    def baseurl(self):
+        return "{0}/sites/{1}/dataAccelerationReport".format(self.parent_srv.baseurl, self.parent_srv.site_id)
+
+    @api(version="3.8")
```

Comment:
Just noticed this here and on the server side. Is this new for 2020.3? 2020.3 is v3.9.

Reply:
Thanks for checking! This is actually for 2020.2. :)